distribution/project-manager/THIRD-PARTY/net.java.dev.jna.jna-5.14.0/NOTICES

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2012 Timothy Wall, All Rights Reserved
```

Comment:
I am pretty sure we shall not be using JNA in `project-manager` in runtime. `project-manager` is _native image_ compiled application and it is using [system Java](https://www.graalvm.org/sdk/javadoc/org/graalvm/nativeimage/c/package-summary.html) and not JNA.

Reply:
Seems to be caused by unfortunately specified dependencies of `scala-libs-wrapper`:
```
project-manager --> scala-yaml --> scala-libs-wrapper --> net.java.dev.jna
```

But `net.java.dev.jna` is excluded from `scala-libs-wrapper.jar`. So it is **not** present in project-manager.